helm/chaos-mesh/values.yaml

```diff
@@ -276,6 +276,8 @@ dashboard:
   env:
     LISTEN_HOST: "0.0.0.0"
     LISTEN_PORT: 2333
+    METRIC_HOST: "0.0.0.0"
+    METRIC_PORT: 8080
```

Comment:
Port `8080` is usually used for the default web service. I don't really agree with exposing metrics-related stuff to this.

Reply:
@STRRL @WangXiangUSTC Any suggestions? For example, `2334`.

Reply:
yes,using 2334 makes it looks better, because even the origin listen port did not use 8080.

Reply:
Done.